src/client/components/landing/_base.js

```diff
@@ -0,0 +1,325 @@
+import {extend, mobileCheck, q, color2Hex} from './helpers.js'
```

Comment:
what does this file do? if its Vanta's code, instead we can just use the minified code `vanta.globe.min.js` file from Vanta GH and put it under `...client/vendor/...` so that we dont have to import the entire source code for vanta?

Reply:
> Also how does it work for the Light/Dark version? will Users be able to toggle or is it hard coded for now?

It should be hard-coded. I was working with @sqphoenix on the color design of the animation, and I posted screenshots of different color combos to help with that.